api/v1alpha3/virtualmachine_types.go

```diff
@@ -159,6 +159,12 @@ const (
 	PVCDiskDataExtraConfigKey = "vmservice.virtualmachine.pvc.disk.data"
 )
 
+const (
+	// VCUUIDAnnotation on a VirtualMachine contains the UUID of the
+	// VMware vCenter (VC) that is managing this virtual machine.
+	VCUUIDAnnotation = GroupName + "/vc-uuid"
```

Comment:
Do we want to call out vCenter as the source of the ID? Why not keep the naming generic, such as:

```go
const (
	// InitialLocationID describes the ID of the location where a
	// VirtualMachine was initially deployed.
	InitialLocationID = GroupName + "/initial-location-id"
)
```

We do not plan to change this when the VM is restored on another platform, correct? We could instead introduce a second annotation:

```go
const (
	// CurrentLocationID describes the unique ID of the location where a
	// VirtualMachine is currently managed.
	CurrentLocationID = GroupName + "/current-location-id"
)
```

Reply:
That's a good point.  Updating.

Reply:
@akutz , Per our conversation, changed the text to "manager-id".  Let me know if this makes sense.